.source/percona-theme/changes/frameset.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Frameset//EN"
+  "http://www.w3.org/TR/html4/frameset.dtd">
+<html>
+  <head>
+    <title>{% trans version=version|e, docstitle=docstitle|e %}Changes in Version {{ version }} &mdash; {{ docstitle }}{% endtrans %}</title>
+  </head>
+  <frameset cols="45%,*">
+    <frame name="main" src="changes.html">
+    <frame name="src" src="about:blank">
```

Comment:
@PaulJacobs-percona Just a sanity check: do we really need that second frame with no content? On purpose I guess?

Reply:
This is part of the to-be-deprecated RST source. And percona-theme is downloaded fresh each build by the makefile and used by all Sphinx projects in Percona (afaik). In other words, I should have said 'ignore everything under .source).

Reply:
Oh, I see. I'd be great to leave an comment `<-- ^^ -->` there :)